.github/workflows/build.yml

```
@@ -0,0 +1,88 @@
+name: OpenWISP CI Build
+
+on:
+  push:
+    branches:
+      - master
+      - dev
+  pull_request:
+    branches:
+      - master
+      - dev
+
+jobs:
+
+  build:
+    name: Python==${{ matrix.python-version }} | ${{ matrix.django-version }}
+    runs-on: ubuntu-latest
+
+    services:
+      redis:
+        image: redis
+        ports:
+          - 6379:6379
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - 3.6
+          - 3.7
+        django-version:
+          - django~=2.2
+          - django~=3.0
+          - django~=3.1
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install system packages
+      run: sudo apt -y install sqlite3 gdal-bin
```

```suggestion
      run: sudo apt-get -y install sqlite3 gdal-bin
```

[`WARNING: apt does not have a stable CLI interface. Use with caution in scripts.`](https://github.com/openwisp/openwisp-notifications/runs/1364607740?check_suite_focus=true#step:5:8)

We can make it less noisy by using the `-qq` flag if you want. Also, do you think we need to run `apt-get update` before as well? It works right now but I'm not sure if something might break in the future.